App/Redux/PhotoViewingSelectors.ts

```diff
@@ -104,7 +104,7 @@ export function getSharedPhotos (state: RootState): ReadonlyArray<SharedPhoto> {
       .filter((photo) => photo.author_id === selfId)
       .map((photo): SharedPhoto => {
         const files = photo.files.length ? photo.files[0] : undefined
-        const original = files && files.links ? files.links.exif.checksum : photo.block
+        const original = files && files.links ? files.links.thumb.checksum : photo.block
```

Comment:
@sanderpick this is the code that's pulling the unique list of shared photos (regardless of thread or comments, likes, etc)

Reply:
Just made this minimal change for now, created #762 to improve